parser_library/test/lexing/logical_line_test.cpp

```diff
@@ -267,9 +267,12 @@ TEST(logical_line, iterator_difference)
     const auto e_1 = std::prev(e);
 
     EXPECT_EQ(std::distance(b, e), e - b);
+    EXPECT_EQ(std::ranges::distance(b, e), e - b);
     EXPECT_EQ(-(b - e), e - b);
     EXPECT_EQ(std::distance(b, l1e), l1e - b);
+    EXPECT_EQ(std::ranges::distance(b, l1e), l1e - b);
     EXPECT_EQ(-(b - l1e), l1e - b);
     EXPECT_EQ(std::distance(b, e_1), e_1 - b);
+    EXPECT_EQ(std::ranges::distance(b, e_1), e_1 - b);
```

Comment:
Is it intentional to have both `std::distance` and `std::ranges::distance` tests? Is it to verify that it works correctly for both algorithms?

Reply:
Yes, it is intentional, to test both algs.